pkg/execute/executor.go

```diff
@@ -80,6 +80,8 @@ const (
 	NotifierStartMsg = "Brace yourselves, notifications are coming from cluster '%s'."
 	// IncompleteCmdMsg incomplete command response message
 	IncompleteCmdMsg = "You missed to pass options for the command. Please run /botkubehelp to see command options."
+	// WrongClusterCmdMsg incomplete command response message
+	WrongClusterCmdMsg = "I am sorry, I was not configured for the cluster '%s'."
```

Comment:
We can come up with a better response. Something like - 
```suggestion
	WrongClusterCmdMsg = "Sorry, the admin hasn't configured me to do that for the cluster '%s'."
```